src/Uno.UI/UI/Xaml/Controls/Button/RadioButton.cs

```diff
@@ -96,7 +96,7 @@ public string GroupName
 				typeof(string), 
 				typeof(RadioButton), 
 				new FrameworkPropertyMetadata(
-					(string)null, 
+					string.Empty, 
```

Comment:
This might a bit trickier. As I have worked on `RadioButton`, I noticed something strange:

```
var radioButton = new RadioButton();
Debug.WriteLine(radioButton.GroupName == ""); // True
Debug.WriteLine(radioButton.GetValue(RadioButton.GroupNameProperty) == null); // True
```

So while the property itself returns empty string, the dependency property has a default value of `null`. I am not sure whether this is just a `RadioButton` oddity, or it may be the case of some other string-based dependency properties. In addition, trying to set `GroupName = null` ends up with an exception.

In my `RadioButton` branch I left the `null` as default value and used `??` in the property getter to return empty string instead of `null`, and throwing in the setter for `null` value.